sdk/src/main/java/software/amazon/awssdk/iot/discovery/DiscoveryClientConfig.java

```diff
@@ -13,6 +13,7 @@
     final private String region;
     final private int maxConnections;
     final private HttpProxyOptions proxyOptions;
+    final private String ggServerName;
 
     public DiscoveryClientConfig(
```

Comment:
(just me ranting, too late to fix it now...)
oye
The idea behind a "config" classes is to let us seamlessly add additional args over time.
But this one has a constructor that takes every value .... at which point why do we have a "config" class at all? We could have just put this giant constructor directly on the `DiscoveryClient` class and not had this `DiscoveryClientConfig` class at all